Add user verification research results (#347)
passkeydeveloper/passkeys.dev: content/docs/reference/windows.md

Persistent linking is available between Android devices (authenticator) and Chrome and Edge (clients) on Windows. iOS and iPadOS do not support persistent linking.

### User Verification Behavior

> When a user tries to interact with a passkey on Windows 11, one of screen lock methods is used for user verification via Windows Hello. Starting in Windows 11 22H2, users must set up Windows Hello with at least a device PIN. Setting up facial recognition or fingerprint recognition are optional.

Reviewer: ```suggestion
When a user tries to interact with a passkey on Windows 11, an available screen unlock method is used for user verification via Windows Hello. Starting in Windows 11 22H2, users must set up Windows Hello with at least a device PIN. Setting up facial recognition or fingerprint recognition are optional.
```

Where these biometrics are not configured or available, both passkey creation and authentication fall back to asking for the Windows Hello PIN.

#### Chrome 120

- When biometrics are not configured on Windows, or not available on the device:

>   - The behavior for both `userVerification='required'` and `userVerification='preferred'` are the same: Windows Hello asks for the device PIN for both passkey creation and authentication. Since they fail locally if user verification fails, the server can always expect the UV flag to be `true`.

Reviewer 2: Same here: 
> Since they fail locally if user verification fails, the server can always expect the UV flag to be `true`.

Author: How does this sound?

> Windows Hello asks for the device PIN for both passkey creation and authentication. Since user verification fails locally, the server only receives a successful response with the UV flag to be `true`.

>   - Calling `PublicKeyCredential.isUserVerifyingPlatformAuthenticator()` returns `true`.

Reviewer: ```suggestion
  - Calling `PublicKeyCredential.isUserVerifyingPlatformAuthenticatorAvailable()` returns `true`.
```
